spec.md

```diff
@@ -321,6 +321,12 @@ The process remains unchanged for chunked upload, except that the post request M
 Content-Length: 0
 ```
 
+If the registry has a minimum chunk size, the response SHOULD include the following header, where `<size>` is the size in bytes:
+
+```
+OCI-Chunk-Min-Bytes: <size>
```

Comment:
...Min-Length: <length>

where length is the minimum size in bytes of a chunk...

Reply:
I was going with "bytes" to avoid needing to parse a string with different units. What's the thought process for changing to "length"?

Reply:
I was just looking for consistency in naming

Reply:
Got it, updated to be Length.